### PR TITLE
feat(superuser): Local Dev settings for superuser access

### PR DESCRIPTION
**Pull request by maxiuyuan**
Devs couldn't get superuser access on local dev as some don't have SSO setup locally. The solution was to add settings to disabled validation of superuser access form and enabled superuser session upon login for local dev

### CODE REVIEW

File: src/sentry/api/endpoints/auth_index.py [hunk 1]
```diff
@@ -111,9 +117,12 @@ def _require_password_or_u2f_check():
 
         if _require_password_or_u2f_check():
             authenticated = self._verify_user_via_inputs(validator, request)
-
+        breakpoint()
```
Comment (nhsiehgit) [inline src/sentry/api/endpoints/auth_index.py]: did you mean to keep this?

Comment (dashed) [inline src/sentry/api/endpoints/auth_index.py]:
```suggestion
```

We probably dont want this in prod :D

Comment (maxiuyuan) [reply src/sentry/api/endpoints/auth_index.py]: no i just got rid of it

Comment (maxiuyuan) [reply src/sentry/api/endpoints/auth_index.py]: yeah i just got rid of it :c I have no idea why but usually that is caught

**Review by nhsiehgit — APPROVED** [review]

**Review by dashed — APPROVED** [review]

**Pull request merged**